source/atlas-cli-deploy-local.txt

```diff
@@ -86,6 +86,105 @@ Before you begin, complete the following prerequisites:
 
 .. include:: /includes/steps-atlas-cli-deploy-local-prereqs.rst
 
+.. _atlas-cli-pvt-registry-setup:
+
+(Optional) Set up a Private Image Registry 
+------------------------------------------
+
+This section demonstrates how to pull and tag images, load images to a
+container registry, and configure the {+atlas-cli+} to use private
+images for building local clusters. 
+
+.. procedure:: 
+   :style: normal 
+
+   .. step:: Install the Docker command line tools.
+   
+   .. step:: Create your container registry with the namespace and name of repository. 
+
+   .. step:: Fetch the MongoDB image for local deployment. 
+
+      To fetch, run the following command after replacing the following
+      placeholder values:
+
+      .. list-table:: 
+         :stub-columns: 1 
+
+         * - ``<version-number>``
+           - The version number of {+atlas-cli+} that you want to use in
+             your deployment. 
+
+         * - ``<container-registry-namespace>`` 
+           - The fully qualified path and name of the container registry
+             namespace that you created in step 2.
+    
+      .. code-block:: shell 
+
+         docker tag mongodb/mongodb-atlas-local:<version-number> <container-registry-namespace>/mongodb-atlas-local:<version-number>
+         
```

Comment:
Here we should provide a note about the different versions of MongoDB, that when you `atlas deployment setup` you get the option of deploying MDB 7 or 8, and that in order for both to work when the CLI pulls from the local registy, the user will need to complete steps 3 and 4 twice; once with the `:7.0` tag and once with the `:8.0` tag.